### ISSUE TITLE
Gathering data from --profile-ir

### ISSUE THREAD

**Issue by AlexandreEichenberger** [onnx/onnx-mlir#2447]
Right now, when running with the --profile-ir option, we get this output
```
alexe@AChampignon simd % onnx-mlir -O3 -march=arm64 bertsquad10-v2.mlir -profile-ir=Onnx -onnx-op-report=Simd
==ONNX-SIMD-REPORT==, onnx.ReduceMeanV13-simd, bert/embeddings/LayerNorm/moments/mean, 16, 768, horizontal
alexe@AChampignon simd % ~/OM/onnx-mlir/build/Debug/bin/run-onnx-lib bertsquad10-v2.so                       
Load model file bertsquad10-v2.so with entry point run_main_graph
Start computing 1 iterations
#0) before onnx.ReduceMeanV13 Time elapsed: 1692207454.743763 accumulated: 1692207454.743763 (bert/embeddings/LayerNorm/moments/mean)
#1) after  onnx.ReduceMeanV13 Time elapsed: 0.000123 accumulated: 1692207454.743886 (bert/embeddings/LayerNorm/moments/mean)
Finish computing 1 iterations
```

Any objections if I were to change the output printed by the tracing as follows?
```
==profiling==, onnx.ReduceMeanV13, bert/embeddings/LayerNorm/moments/mean, before, 1692207454.743763, 1692207454.743763
==profiling==, onnx.ReduceMeanV13, bert/embeddings/LayerNorm/moments/mean, after, 0.000123, 1692207454.743886
```

Namely using this pattern
```
==profiling==, <op name>, <node name>, (before | after), <elapsed time>, <accumulated time>
```

That way, I could more easily combine the data I track in `make-report.pl` to include runtime, as well as easily processing the traces as well and summarizing per op-type, per node name,...

The idea is to kind of normalize the patterns from all our profiling places,  compiler, runtime exec,... to a single format.
All of them currently use `<type of analysis>, <op name>, <node name>,` followed by comma separated fields specific to the analysis

**Comment by tungld** [comment]
I have no objection.

**Comment by AlexandreEichenberger** [comment]
Implemented in #2451
